internal/k8s/client.go

```diff
@@ -13,11 +11,11 @@ import (
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/browser"
 	"github.com/windmilleng/tilt/internal/logger"
-	"github.com/windmilleng/tilt/internal/output"
 )
 
 type PodID string
 type ContainerID string
```

Comment:
...it occurs to me that this is actually a Docker thing rather than a k8s thing... ugh we should have a Docker package where this can live...